hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/safemode/TestSCMSafeModeManager.java

```diff
@@ -314,6 +321,16 @@ public void testSafeModeExitRuleWithPipelineAvailabilityCheck(
 
     List<Pipeline> pipelines = pipelineManager.getPipelines();
 
+    for (Pipeline pipeline : pipelines) {
+      List<DatanodeDetails> nodes = pipeline.getNodes();
+      for (DatanodeDetails node : nodes) {
+        SCMDatanodeProtocolServer.NodeRegistrationContainerReport nodeRegistrationContainerReport =
+            new SCMDatanodeProtocolServer.NodeRegistrationContainerReport(node, null);
+        queue.fireEvent(SCMEvents.NODE_REGISTRATION_CONT_REPORT,
+            nodeRegistrationContainerReport);
+      }
+    }
+
```

Comment:
This is not required in this test currently.

Reply:
Thank you for your suggestion! I have removed this part of the code.